src/python/RucioUtils.py

```diff
@@ -116,25 +116,34 @@ def getRuleQuota(rucioClient=None, ruleId=None):
 
 def getRucioUsage(rucioClient=None, account=None, activity =None):
     """ size of Rucio usage for this account (if provided) or by activity """
-    filters = {'activity': activity}
-    
-    if account is not None and account != 'crab_tape_recall':
-        filters['account'] = account
-    
-    rules = rucioClient.list_replication_rules(filters=filters)
-
-    if activity == 'Analysis Input':
-        valid_states = ['OK', 'REPLICATING', 'STUCK', 'SUSPENDED']
-    elif activity == 'Analysis TapeRecall':
-        valid_states = ['REPLICATING', 'STUCK', 'SUSPENDED']  # Exclude 'OK' state
+    if activity is None:
+        if account is None:
+            print("Error: Account and Activity both unspecified")
```

Comment:
we should not use print() statement for code which can be used in services like TW, where stdout is basically lost. Either return 0 (if it will be correctly used by the caller) or raise an Exception (e.g. here both arguments will be None only due to some bugs) or add a logger as argument and use logging.

Reply:
I can change it to raise ValueError